bootstrap/000_setup_base_tools.sql

```diff
@@ -16,6 +32,19 @@ BEGIN
       INSERT (key, value) VALUES (source.key, source.value);
 END;
 
+CREATE OR REPLACE PROCEDURE internal.get_config(key string) RETURNS STRING LANGUAGE SQL
+    AS
+BEGIN
+    let cnt number := (SELECT COUNT(*) AS cnt FROM internal.config WHERE key = :key);
```

Comment:
I don't think this check is needed. Snowflake will return `null` if key is not found

Reply:
good point. The query would return 0 row when key is not found, but the assignment `let config_value string := (query)` would get `config_value = null`.  Removed the check.